homeassistant/components/vicare/__init__.py

```diff
@@ -55,6 +56,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up from config entry."""
     _LOGGER.debug("Setting up ViCare component")
 
+    await _async_migrate_unique_ids(hass, entry)
```

Comment:
Does this make the PR a breaking change? For example, do automations, scenes, etc need to be upodated after this change?

Reply:
I thought by providing a migrate function it's non-breaking. But you're right, scripts and such would still fail. Basically it would not match the entity base class if it's not changed and naming would be inconsistent.

Reply:
One option is to use the `repair` functionality. This will notify the user of the need to change the dependent items.

Reply:
That sounds like a good idea, will draft this.